printers/prusa-mk3s/steppers.cfg

```diff
@@ -0,0 +1,33 @@
+# WARNING. DO NOT EDIT THIS FILE.
+# To override settings from this file, you can copy and paste the relevant
+# sections into your printer.cfg and change it there.
+
+[stepper_x]
+step_pin: x_step_pin
+dir_pin: !x_dir_pin
+enable_pin: !x_enable_pin
+microsteps: 32
```

Comment:
Change all microstep settings to 64 for noise reduction when interpolation is off.

Reply:
There is no way to use it with 64 microsteps, maximum is 32

![image](https://user-images.githubusercontent.com/7290345/221525362-ac59f866-69d8-4175-a724-5b0fb98e6917.png)